src/InTextAnnotationParser.php

```diff
@@ -141,6 +141,64 @@ public function parse( &$text ) {
 	}
 
 	/**
+	 * @since 2.4
+	 *
+	 * @param string $text
+	 *
+	 * @return text
+	 */
+	public static function obscureAnnotation( $text ) {
```

Comment:
![3326873_orig](https://cloud.githubusercontent.com/assets/146040/11873542/599330d0-a4db-11e5-8edf-cef54fa2e948.jpg)

Production usage of this new method is not in this PR, so it's hard to say much more about this